src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapperTest.java

```diff
@@ -2418,6 +2413,36 @@ void should_map_task_role_permissions_for_granular_permissions() {
         assertTrue(taskRolePermissions.getAuthorisations().contains("STANDARD"));
     }
 
+    @Test
+    void reconfigure_config_additional_attributes_dmn_fields() {
+        TaskResource taskResource = createTaskResource();
+        List<ConfigurationDmnEvaluationResponse> additionalPropertyDMNResponse = asList(
+                new ConfigurationDmnEvaluationResponse(stringValue("additionalProperties_roleAssignmentId"),
+                        stringValue("roleAssignmentId"), booleanValue(true)
+                ),
+                new ConfigurationDmnEvaluationResponse(stringValue("additionalProperties_key1"), stringValue("value1"),
+                        booleanValue(false)
+                ),
+                new ConfigurationDmnEvaluationResponse(stringValue("additionalProperties_key2"), stringValue("value2"),
+                        null
+                ),
+                new ConfigurationDmnEvaluationResponse(stringValue("additionalProperties_key3"), stringValue(""),
+                        booleanValue(true)
+                ),
+                new ConfigurationDmnEvaluationResponse(stringValue("additionalProperties_key4"), stringValue(null),
+                        booleanValue(true)
+                )
```

Comment:
would adding 2 additional edge cases when - canReconfigure is false and value is empty or null help show we have all edge cases covered?